### DISCUSSION TITLE
Question: Is it recommended to use Task.Result in Orchestrations?

### DISCUSSION THREAD

**Discussion by bhuvesh321** [Azure/azure-functions-durable-extension#1761] [Q&A] [answered]

**Comment by cgillum** [comment, marked as answer]
It’s fine to use Task.Result. I can’t think of many times you need to use this, but it’s not a problem if you do.

**Comment by ConnorMcMahon** [comment]
It's worth noting we have issue #201 about not using `.Result`, but looking at the issue, that seems to be more with repeatedly iterating over an `IEnumerable<Task>`, than the actual calling of `.Result` on tasks..
